src/sentry/static/sentry/app/views/settings/components/forms/radioField.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import InputField from './inputField';
+import {growIn} from './styled/animations';
+
+class RadioField extends InputField {
+  static propTypes = {
+    id: PropTypes.number,
+    value: PropTypes.string,
+  };
+
+  onChange = (value, onChange, onBlur, e) => {
+    onChange(value, e);
+    onBlur(value, e);
+  };
+
+  isSelected = ({value, id}) => {
+    return value ? value === id : id === 0;
+  };
+
+  render() {
+    return (
+      <InputField
+        {...this.props}
+        field={({onChange, onBlur, value, disabled, ...props}) => (
+          <div>
+            {(props.choices() || []).map((choice, index) => {
+              const {id, name} = choice;
```

Comment:
We don't have any style rules for this, but I like to destructure in the function parameter (so I don't have to come up with a name):

```
{(props.choices() || []).map(({id, name}, index) => {
```